tools/cp-caps/README.md

```diff
@@ -54,26 +54,28 @@ Show help info.
 
 ```bash
 $ python3 ./tools/cp-caps/rcp_caps_test.py -h
-usage: rcp_caps_test.py [-h] [-c] [-d] [-p] [-t] [-v] [-D]
+usage: rcp_caps_test.py [-h] [-c] [-l] [-d] [-p] [-t] [-v] [-D] [--frame-format]
 
 This script is used for testing RCP capabilities.
 
 options:
   -h, --help           show this help message and exit
   -c, --csl            test whether the RCP supports CSL transmitter
-  -d, --diag-commands  test whether the RCP supports all diag commands
   -l, --link-metrics   test whether the RCP supports link metrics
+  -d, --diag-commands  test whether the RCP supports all diag commands
   -p, --data-poll      test whether the RCP supports data poll
-  -t, --throughput     test the Thread network 1-hop throughput
+  -t, --throughput     test Thread network 1-hop throughput
   -v, --version        output version
   -D, --debug          output debug information
+  --frame-format       test whether the RCP supports 802.15.4 frames of all formats
```

Comment:
nit: Maybe
- group this line with other test cases
- introduce a short form argument (e.g. -f)